scripts/qa_crowbarsetup.sh

```diff
@@ -1525,7 +1525,7 @@ EOF
         /opt/dell/bin/json-edit -a attributes.network.networks.public.ranges.host.end -v 10.162.211.191 $netfile
     fi
     # Setup network attributes for custom MTU
-    if [[ $want_mtu_size -ne 1500 ]]; then
+    if [[ $want_mtu_size ]] && [[ $want_mtu_size -ne 1500 ]]; then
```

Comment:
@bmwiedemann @jdsn I removed now the want_mtu_size from mkcloud to eliminate the duplicate default value. I also removed the check so that the mtu is always written.

Thanks for your comments :)